README.md

```diff
@@ -1,742 +1,83 @@
-    </system.webServer>
-    ```
-
-3. Configure OpenTelemetry in your application startup:
-    ```csharp
-    public class WebApiApplication : HttpApplication
-    {
-        private TracerFactory tracerFactory;
-
-        protected void Application_Start()
-        {
-            this.tracerFactory = TracerFactory.Create(builder =>
-            {
-                builder
-                    .UseJaeger(c =>
-                    {
-                        c.AgentHost = "localhost";
-                        c.AgentPort = 6831;
-                    })
-                    .AddRequestAdapter()
-                    .AddDependencyAdapter();
-            });
-        }
-
-        protected void Application_End()
-        {
-            this.tracerFactory?.Dispose();
-        }
-    }
-    ```
-
-### Using StackExchange.Redis adapter
-
-Outgoing http calls to Redis made using StackExchange.Redis library can be automatically tracked.
-
-1. Install package to your project:
-   [OpenTelemetry.Adapter.StackExchangeRedis][OpenTelemetry-adapter-stackexchange-redis-nuget-url]
-
-2. Configure Redis adapter
+The .NET [OpenTelemetry](https://opentelemetry.io/) client.
 
-    ```csharp
-    // connect to the server
-    var connection = ConnectionMultiplexer.Connect("localhost:6379");
+## Installation
 
-    using (TracerFactory.Create(b => b
-                .SetSampler(new AlwaysSampleSampler())
-                .UseZipkin(options => {})
-                .SetResource(Resources.CreateServiceResource("my-service"))
-                .AddAdapter(t =>
-                {
-                    var adapter = new StackExchangeRedisCallsAdapter(t);
-                    connection.RegisterProfiler(adapter.GetProfilerSessionsFactory());
-                    return adapter;
-                })))
-    {
+This repository includes multiple installable packages. The `OpenTelemetry.Api`
+package includes abstract classes and no-op implementations that comprise the OpenTelemetry API following
+[the
+specification](https://github.com/open-telemetry/opentelemetry-specification).
+The `OpenTelemetry` package is the reference implementation of the API.
 
-    }
-    ```
+Libraries that produce telemetry data should only depend on `OpenTelemetry.Api`,
+and defer the choice of the SDK to the application developer. Applications may
+depend on `OpenTelemetry` or another package that implements the API.
 
-You can combine it with dependency injection as shown in previous example.
+**Please note** that this library is currently in _alpha_, and shouldn't
+generally be used in production environments.
 
-### Custom samplers
+The API and SDK packages are available on the following NuGet feeds:
+* [MyGet V2](https://www.myget.org/F/opentelemetry/api/v2)
+* [MyGet V3](https://www.myget.org/F/opentelemetry/api/v3/index.json)
 
-You may configure sampler of your choice
+To install the development versions of these packages instead, clone or fork this repo and do an editable install:

```

Comment:
Wouldn't this better on the contributing doc since it is typically used by developers?

Reply:
Good point.

Reply:
I've removed this line.
Will send a separate PR to update / clean up the `CONTRIBUTING.md`.